### PR TITLE
[feature] Add environment type

### PR DESCRIPTION
**Pull request by sirNikolai**
### Purpose of pull request

**Related Issue**: #16

Add environment variable to prevent running into client.login issue

### Pull request checklist

- [x] Branch and PR are named correctly
- [ ] Updated relevant documentation
- [x] Tested with a tester bot
- [ ] Wrote unit tests for changes

### Testing instructions

Local testing: would need to set `ENV=testing` in .env

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -30,7 +30,9 @@ client
   .on('warn', (warn) => client.logger.warn(warn));
 
 // Login client
-client.login(BOT_TOKEN);
+if(ENV !== 'testing') {
```
Comment (brendacs) [inline src/client.js:33]:
```suggestion
if (ENV !== 'dev') {
```

File: src/constants.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import dotenv from 'dotenv';
 
 // Environment variables
 dotenv.config();
-export const { BOT_TOKEN, MONGODB } = process.env;
+export const { BOT_TOKEN, MONGODB, ENV } = process.env;
```
Comment (brendacs) [inline src/constants.js:5]: Alpha order required

**Review by brendacs — CHANGES_REQUESTED** [review]
Nice! Just minor comments

File: src/client.js (hunk 1 above)
Comment (sirNikolai) [inline src/client.js:33]: possibly should add variables in constants but I was thinking 3 vars:
- testing (for unit testing)
- development
- production 

Not sure on diff of last two but possibly something that will be needed in the future.

Comment (brendacs) [reply src/client.js:33]: Yeah, I like this

**Review by brendacs — APPROVED** [review]

**Pull request merged**